mmv1/third_party/validator/tests/data/example_compute_instance.json

```diff
@@ -68,5 +68,20 @@
         }
       }
     }
+  },
+  {
+    "name": "//iam.googleapis.com/projects/{{.Provider.project}}/serviceAccounts/placeholder-unique-id",
```

Comment:
not quite sure about this placeholder-unique-id part. If it can resolve properly, it should be more meaningful than this?

Reply:
This `placeholder-unique-id` part is replaced during tests execution by the function `normalizeAssets`.
This function seems to replace every asset name matching the regexp `/placeholder-[^/]+` with `placeholder-foobar`

https://github.com/GoogleCloudPlatform/magic-modules/blob/6f779a907f2ef6f59e7acf52ee0f2e135ef1bbec/mmv1/third_party/validator/tests/source/read_test.go.erb#L74

https://github.com/GoogleCloudPlatform/magic-modules/blob/4c6fe86dcd980b210bfddba33148fb12f3b26ae7/mmv1/third_party/validator/tests/source/init_test.go#L126 

Therefore, using an assetName with `placerholder-*` ensures that the tests can worked successfully 
Let me in case i misunderstood something

Reply:
the placeholder only makes sense if the resource name is only known after creation, if we have a test case which is just for existing resource, the asset name should not have a placeholder in theory.

Reply:
it looks like the update test includes a non-placeholder value 👍